pennylane/workflow/qnode.py

```diff
@@ -495,16 +495,8 @@ def __init__(
                 gradient_kwargs,
             )
 
-        if expansion_strategy is not None:
-            warnings.warn(
-                "The 'expansion_strategy' attribute is deprecated and will be removed  "
-                "in version 0.39. For full control over the stage to which the tape is "
-                "constructed, use the 'pennylane.workflow.construct_batch' function.",
-                qml.PennyLaneDeprecationWarning,
-            )
-
-        # Default to "gradient" to maintain default behaviour of "draw" and "specs"
-        expansion_strategy = expansion_strategy or "gradient"
+        if isinstance(device, qml.devices.LegacyDevice):
+            device = qml.devices.LegacyDeviceFacade(device)

```

Comment:
This is problematic because the instance check will return `True` whether `device` is of type `LegacyDeviceFacade` or `LegacyDevice` due to the new metaclass.

Reply:
Resolved offline.